pims/base_frames.py

```diff
@@ -606,3 +606,28 @@ def __repr__(self):
             s += "Axis '{0}' size: {1}\n".format(dim, self._sizes[dim])
         s += """Pixel Datatype: {dtype}""".format(dtype=self.pixel_type)
         return s
+
+    def _init_axis_if_exists(self, axis, size, min_size=1):
```

Comment:
I think this function does not belong in the baseclass, the logic of how to init axes is really reader-specific.

Utils functions tend to grow out of hand in my experience. Just hardcode the logic if its simple.